expo-package/v2.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * In Expo 34, all the modules such as image picker, document picker, permissions etc etc have been moved to their own packages.
+ * And they have been removed from Expo package. But these packages don't work with Expo 32 sdk (because of linking issue).
+ */
+import { registerNativeHandlers } from 'stream-chat-react-native-core';
+import NetInfo from '@react-native-community/netinfo';
+import { Constants } from 'react-native-unimodules';
+import * as Expo from 'expo';
+
+let ImagePicker;
+let DocumentPicker;
+let Permissions;
+let manifest = {};
+
+manifest = Constants.manifest;
+if (!manifest) {
+  manifest = Expo.Constants.manifest;
+}
+
+if (manifest.sdkVersion.split('.')[0] >= 33) {
+  ImagePicker = require('expo-image-picker');
+  Permissions = require('expo-permissions');
+  DocumentPicker = require('expo-document-picker');
+} else {
+  ImagePicker = Expo.ImagePicker;
+  Permissions = Expo.Permissions;
+  DocumentPicker = Expo.DocumentPicker;
```

Comment:
we can remove this I guess ... now that we don't support expo < 39

Reply:
sure?